test/common/testlib.py

```diff
@@ -766,6 +766,16 @@ def is_nondestructive(self):
         test_method = getattr(self.__class__, self._testMethodName)
         return getattr(test_method, "_testlib__non_destructive", False)
 
+    @staticmethod
+    def disable_preload(machine, *packages):
+        for pkg in packages:
+            machine.write("/usr/share/cockpit/%s/override.json" % pkg, '{ "preload": [ ] }')
+
+    @staticmethod
+    def enable_preload(machine, package, *pages):
+        machine.write("/usr/share/cockpit/%s/override.json" % package, '{ "preload": [%s]}' %
+                      ', '.join('"{0}"'.format(page) for page in pages))
```

Comment:
Seems easier and more robust to just remove the override files again?

Reply:
Well, that counts on that file existing (so we would need to be careful when enabling some preload that we disabled it before). Also this can be used to enable it on packages that don't preload by default.